### ISSUE TITLE
Provide examples app as apk to avoid annoying build process

### ISSUE THREAD

**Issue by carstenhag** [mapbox/mapbox-navigation-android#3348]
Hello there! I have been trying to use the new 1.0.0-rc2 version. As I could not figure out how it works with the sample code, I wanted to check out the example app.

But the thing is, instead of only installing the APK and looking at the corresponding file, I now need to set up the entire mapbox-navigation-android project on my machine.

Now with a normal project this shouldn't be too terrible either, but bintray requires a username and password for the plugins dependency, apparently.

tl;dr please provide apk for examples app

**Comment by carstenhag** [comment]
Not stale
